### DISCUSSION TITLE
Blog comment on Creating a playful new identity for Weaviate

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1283] [Docs/blog comments]
# Blog comment on Creating a playful new identity for Weaviate

In January 2023, SeMI Technologies became Weaviate as we aligned...

https://weaviate.io/company/playbook/creating-a-playful-new-identity-for-weaviate

**Comment by michaelpoley** [comment]
Great documentation - the 3d logo really elevates the brand. Thanks for sharing your design process!
